generator/_config.yml

```diff
@@ -0,0 +1,101 @@
+# SITE CONFIGURATION
+#baseurl: / # the subpath of your site, e.g. /blog
+url: "0.0.0.0" # the base hostname & protocol for your site
+#destination: /srv/jekyll/_site
+keep_files: [./*]
+
+# THEME-SPECIFIC CONFIGURATION
+theme_settings:
+  # Meta
+  title: Hyperledger Sawtooth
+  avatar: avatar.png
+  gravatar: # Email MD5 hash
+  description: "A website with blog posts and pages" # used by search engines
+
+  # Header and footer text
+  header_text: >
+     <h1>Sawtooth</h1>
+  header_text_feature_image: ../examples/img/hero-bg.jpg
+  footer_text: >
+    <h2>Sawtooth</h2>
+    <p>
+      <a href="https://chat.hyperledger.org/channel/sawtooth" target="_blank" title="Rocket Chat">RocketChat</a>
+    </p>
+    <p>
+      <a href="/docs/" target="_blank" title="Documentation">Documentation</a>
+      &bull;
+      <a href="https://github.com/hyperledger/sawtooth-core/" target="_blank" title="Github">Github</a>
+      &bull;
+      <a href="/examples/seafood.html" title="Seafood Supply Chain Traceability">Seafood Supply Chain Traceability</a>
+      &bull;
+      <a href="/examples/bond.html" title="Bond Asset Settlement">Bond Asset Settlement</a>
+      &bull;
+      <a href="/examples/marketplace.html" title="Marketplace Digital Asset Exchange">Marketplace Digital Asset Exchange</a>
+    </p>
+    <p>
+      <a href="http://www.linuxfoundation.org/privacy" target="_blank" title="Privacy Policy">Privacy Policy</a>
+      &bull;
+      <a href="http://www.linuxfoundation.org/terms" target="_blank" title="Terms of Service">Terms of Service</a>
+    </p>
+    <p>
+      <small class="copyright">&copy; Copyright 2015&ndash;2017, Intel Corporation. Sawtooth is an open source project that is available to the public.</small>
```

Comment:
This is somewhat inaccurate. Per current discussion in #sawtooth-governance, replace with "Copyright &copy; 2015-2018 contributors to Hyperledger Sawtooth". (If we end up with something else, we will just change it to the new thing).